video/out/hwdec/hwdec_vaapi.c

```diff
@@ -87,6 +88,14 @@ static const struct va_create_native create_native_cbs[] = {
 #endif
 };
 
+static bool is_wldmabuf(struct ra* ra){
+#if HAVE_VAAPI_WAYLAND
```

Comment:
Should be `HAVE_DMABUF_WAYLAND` (same for the other one).

Reply:
yes, fixed now.